src/Stripe.net/Services/Account/AccountSharedOptions.cs

```diff
@@ -43,32 +34,15 @@ public abstract class AccountSharedOptions : BaseOptions
         [JsonProperty("external_account")]
         public AccountBankAccountOptions ExternalBankAccount { get; set; }
 
-        [JsonProperty("legal_entity")]
-        public AccountLegalEntityOptions LegalEntity { get; set; }
-
+        // TODO: Figure out how to handle Individual
```

Comment:
What's the issue exactly? Can't we declare it as `public Person Individual { get; set; }`?

Reply:
Well that works on the resource, not on the parameters. We could use `PersonCreateOptions` maybe but one is a `BaseOptions` while here we'd want an `INestedOptions`

Reply:
I don't think that's a huge problem. The only practical difference between `BaseOptions` and `INestedOptions` is that classes that inherit the former get `AddExpand()` and `AddExtraParam()` methods. It's not a big deal if we use a `BaseOptions` class as a nested options class.

Reply:
I had no idea! Fixed.